zk-collectd.py

```diff
@@ -84,6 +84,12 @@ def _get_mntr_stats(self):
         response = self._send_cmd(MNTR_CMD)
         result = {}
 
+        # If instance stops serving requests (e.g. loses quorum) it still returns "imok" to ruok query but will
+        # force close any client connections and return an error string to all other 4 letter commands. In this
+        # situation we should override service health metric as it's definitely not in a healthy state.
```

Comment:
Could you also comment that this will override the value read in `_get_health_stat()`?

(You also probably need to wrap that comment to 80 columns; make sure you run `flake8` on the file)